admin/src/bench/mod.rs

```diff
@@ -252,6 +294,19 @@ pub fn run_internal(args: Args, db: Db) {
 		0
 	};
 
+	let mut pool = SizePool::from_histogram(sizes::KUSAMA_STATE_DISTRIBUTION, args.uniform);
+	if args.uniform {
+		println!("Generating uniform keys.");
```

Comment:
I am not too sure when we hit non cached keys, but good to have the code for it.

Reply:
Yeah. Reader threads when appending will hit them. Also, writer threads when pruning and appending can hit them sometimes. Might have missed something else.